sources/org.osbuild.ostree

```diff
@@ -107,10 +111,13 @@ class OSTreeSource(sources.SourceService):
         if not gpg:
             remote_add_args = ["--no-gpg-verify"]
 
+        if remote.get("contenturl", ""):
+            remote_add_args.append("--set \"contenturl=%s\"" % remote['contenturl'])
+
         if remote.get("secrets", {}).get("name") == "org.osbuild.rhsm.consumer":
             secrets = Subscriptions.get_consumer_secrets()
-            remote_add_args.append("--tls-client-key-path=%s", secrets['consumer_key'])
-            remote_add_args.append("--tls-client-cert-path=%s", secrets['consumer_cert'])
+            remote_add_args.append("--set \"tls-client-key-path=%s\"" % secrets['consumer_key'])
+            remote_add_args.append("--set \"tls-client-cert-path=%s\"" % secrets['consumer_cert'])
```

Comment:
How did we not see this? Whoops!

Anyway, in all 3 cases you likely want to use f-strings to make pylint happy: `f"--set.... {secrets['consumer_key']}"`